tsconfig.json

```diff
@@ -15,8 +14,6 @@
 
     "noFallthroughCasesInSwitch": true,
     "noUncheckedIndexedAccess": true,
-    "noPropertyAccessFromIndexSignature": false, // temporirly disabled, should be enabled (only 25 errors, easily fixable)
-    "noImplicitReturns": false, // temporirly disabled, should be enabled (only 5 errors, easily fixable)
```

Comment:
Unlike strict rules which are defaulted to true with the `"strict": true`, here their default values are `false`, so you should leave the rules to make sure they are checked at compile time.

Reply:
Oops sorry, that's now fixed :)